javascript-data-model/order-history.js

```diff
@@ -0,0 +1,78 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    total: '$34.00',
+    shipTo: 'JS Masher',
+    orderNumber: '114-3941689-8772232',
+    deliveryDate: 'Delivered Aug 8, 2020',
+    message: '',
+    title: 'JavaScript for impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    return: 'Return window closed on Sep 7, 2020',
+    cost: '$31.55',
+    orderDetails: '',
+    invoice: ''
+  },
+
+  {
+    orderPlaced: 'July 19, 2020',
+    total: '$44.53',
+    shipTo: 'JS Masher',
+    orderNumber: '113-9984268-1280257',
+    deliveryDate: 'Delivered Jul 20, 2020',
+    message: 'Your package was delivered. It was handed directly to a resident.',
+    title: 'The Timeless Way of Building',
+    author: 'Alexander, Christopher',
+    return: 'Return window closed on Aug 19, 2020',
+    cost: '$41.33',
+    orderDetails: '',
+    invoice: ''
+  },
+
+  {
+    orderPlaced: 'July 4, 2020',
+    total: '$17.22',
+    shipTo: 'JS Masher',
+    orderNumber: '114-2875557-9059409',
+    deliveryDate: 'Delivered Jul 7, 2020',
+    message: 'Your package was delivered. It was handed directly to a resident.',
+    title: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+    author: '',
+    return: 'Return window closed on Aug 5, 2020',
+    cost: '$15.98',
+    orderDetails: '',
+    invoice: ''
+
+  },
+
+  {
+    orderPlaced: 'July 3, 2020',
+    total: '$138.93',
+    shipTo: 'JS Masher',
+    orderNumber: '113-2883177-2648248',
+    deliveryDate: 'Delivered Jul 5, 2020',
+    message: '',
+    title: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+    author: '',
+    return: 'Return window closed on Aug 4, 2020',
+    cost: '$94.95',
+    orderDetails: '',
+    invoice: ''
+  },
+
+  {
+    orderPlaced: 'July 3, 2020',
+    total: '$138.93',
+    shipTo: 'JS Masher',
+    orderNumber: '113-2883177-2648248',
+    deliveryDate: 'Delivered Jul 5, 2020',
+    message: '',
+    title: 'The Art of Sql',
+    author: 'Faroult, Stephane',
+    return: 'Return window closed on Aug 4, 2020',
+    cost: '$33.99',
+    orderDetails: '',
+    invoice: ''
+  }
```

Comment:
For this portion, there is no way for me to know that these two items were in the same order. The main object being stored in the `orderHistory` array should be describing the order itself, and then you can store details about each item ordered in a property on that object

Reply:
![Screen Shot 2021-01-26 at 12 36 06 PM](https://user-images.githubusercontent.com/74885918/105902163-20534b80-5fd3-11eb-93e3-0f5f79f474cb.png)